api/sql/ddl/table.sql

```diff
@@ -77,6 +77,15 @@ create table if not exists project(
     nome varchar
 );
 
+create table if not exists projects(
+    prj_id serial primary key,
+    "name" varchar(255) unique not null,
+    description varchar(255),
+    active bool default true,
+    insert_date timestamp default now(),
+    expire_date timestamp
+)
```

Comment:
Como a @taniacruzz havia mencionado anteriormente
```suggestion
);
```